flocker/apiclient/test/test_client.py

```diff
@@ -496,6 +496,29 @@ def got_result(container):
             d.addCallback(got_result)
             return d
 
+        def test_container_state(self):
+            """
+            ``list_containers_state`` returns information about state.
+            """
+            expected, d = create_container_for_test(self, self.client)
+
+            d.addCallback(lambda _ignored: self.synchronize_state())
+
+            d.addCallback(lambda _ignored: self.client.list_containers_state())
+
+            d.addCallback(
+                lambda containers: [(c.node_uuid, c.name) for c in containers]
+            )
+
+            d.addCallback(
+                lambda containers: self.assertIn(
+                    (expected.node_uuid, expected.name),
+                    containers
```

Comment:
:grey_question: No action required: Looks like we don't technically have test coverage of the image and running attributes, but risk seems pretty low (we should get attribute errors in the test below if the server doesn't actually return a JSON blob with those keys I think?). Seems like an okay state of the world to me, just wanted to call it out in case you had a clever idea of how to add test coverage and just forgot to come back and add it or something.